### PR TITLE
Spelling: Language rework

### PR DESCRIPTION
**Pull request by comradekingu**
What does "Download badges" mean?
Also, is "Global" program-wide?

### CODE REVIEW

**Comment by inorichi** [comment]
> What does "Download badges" mean?

It's an indicator to show the number of chapter downloaded, displayed in the top-left edge of a cover with an overlay.

> Also, is "Global" program-wide?

Do you mean this? ` <string name="action_global_search">Global search</string>` This is another feature to search in the whole list of catalogues instead of just one.

**Comment by comradekingu** [comment]
@inorichi Are "Download indicators" OK?
"Search whole list" seems more descriptive.

**Comment by comradekingu** [comment]
@inorichi Do you want me to make the string changes directly (in Weblate)?

**Comment by inorichi** [comment]
Sorry for the late reply. I asked back in the day regarding this PR in our discord (I'm not native) and I agree with most of them but there were a few nitpicks.

Among them:
* We preferred to keep "free and open source" instead of "libre", even if the latter combines the other two.
* Display isn't about the screen, but how to display items (as list, as grid, with/out download badges...)

I think it was @wopian who replied that day but I don't remember.

But yeah, I think it's better to include this with the translations PR (Weblate), although with the english variant I'd like at least another reviewer, as this is the base translation and should provide an accurate meaning.

**Comment by comradekingu** [comment]
@inorichi like so? Better to get this done sooner, as it avoids things being translated twice.

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Features include:
 * Create backups locally or to your cloud service of choice
 
 ## Download
-Get the app from our [releases page](https://github.com/inorichi/tachiyomi/releases) or from [f-droid](https://f-droid.org/packages/eu.kanade.tachiyomi/).
+Get the app from our [releases page](https://github.com/inorichi/tachiyomi/releases) or from [F-Droid](https://f-droid.org/packages/eu.kanade.tachiyomi/).
```
Comment (wopian) [inline README.md]: This needs re-basing - F-Droid is no longer in the README

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -75,7 +75,7 @@
     <string name="action_open_in_browser">Open in browser</string>
     <string name="action_add_to_home_screen">Add to home screen</string>
     <string name="action_display_mode">Change display mode</string>
-    <string name="action_display">Display</string>
+    <string name="action_display">Screen</string>
```
Comment (wopian) [inline app/src/main/res/values/strings.xml]: This is for changing the layout displayed (Grid vs List), as @inorichi mentioned in https://github.com/inorichi/tachiyomi/pull/1367#issuecomment-483713643

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -391,14 +391,14 @@
...
-    <string name="url_not_set">Manga url is not set please click title and select manga again</string>
+    <string name="url_not_set">Manga URL not set, please click title and select manga again</string>
 
     <!-- Category activity -->
     <string name="error_category_exists">A category with this name already exists!</string>
     <string name="snack_categories_deleted">Categories deleted</string>
 
     <!-- Dialog option with checkbox view -->
-    <string name="dialog_with_checkbox_remove_description">This will remove the read date of this chapter. Are you sure?</string>
+    <string name="dialog_with_checkbox_remove_description">This will remove the reading date of this chapter. Are you sure?</string>
```
Comment (wopian) [inline app/src/main/res/values/strings.xml]: `Read` is past tense here, as the date is applied after you have finished reading the chapter.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -420,9 +420,9 @@
     <string name="chapter_subtitle">Chapter %1$s</string>
     <string name="no_next_chapter">Next chapter not found</string>
     <string name="no_previous_chapter">Previous chapter not found</string>
-    <string name="decode_image_error">Image could not be loaded.\nTry changing the image decoder or with one of the options below</string>
+    <string name="decode_image_error">Could not load image.\nTry changing the image decoder, or use one of the options below:</string>
     <string name="confirm_update_manga_sync">Update last chapter read in enabled services to %1$d?</string>
-    <string name="confirm_set_image_as_cover">Do you want to set this image as the cover?</string>
+    <string name="confirm_set_image_as_cover">Use this image as coverart?</string>
```
Comment (wopian) [inline app/src/main/res/values/strings.xml]: cover art

**Comment by hXtreme** [comment]
> "Search whole list" seems more descriptive.

How about "Search everywhere"?

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -306,8 +307,8 @@
     <string name="password">Password</string>
     <string name="show_password">Show password</string>
     <string name="login">Login</string>
-    <string name="login_success">Login successful</string>
-    <string name="invalid_login">Login error</string>
+    <string name="login_success">Now logged in</string>
```
Comment (hXtreme) [inline app/src/main/res/values/strings.xml]: How about, "Successfully logged in"?

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -140,7 +140,8 @@
     <string name="charging">Charging</string>
     <string name="pref_update_only_non_completed">Only update ongoing manga</string>
     <string name="pref_auto_update_manga_sync">Sync chapters after reading</string>
-    <string name="pref_theme">Application theme</string>
+    <string name="pref_ask_update_manga_sync">Confirm before updating</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml]: I think I handled the conflict wrong here, this should've been removed. I'll need to see if I can just update this PR directly.

**Pull request merged**